clash-prelude/src/Clash/XException/Internal.hs

```diff
@@ -53,6 +53,9 @@ import           GHC.Show            (appPrec)
 import           GHC.Stack           (HasCallStack)
 import           System.IO.Unsafe    (unsafeDupablePerformIO)
 
+-- $setup
+-- >>> import Clash.Prelude
```

Comment:
Should this module have `:m -Prelude`?

Reply:
It doesn't seem to ambiguous definitions, so not necessarily :smile: